src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -9,180 +9,366 @@
 -->
 
 <template>
-  <div class="quick-navigation-modal">
-    <div class="container">
+  <GenericModal
+    :showClose="false"
+    :visible="showQuickNavigationModal"
+    :isFullscreen="true"
+    :styleProps="modalStyleProps"
+    @update:visible="$emit('update:showQuickNavigationModal', false)"
```

Comment:
You could replace these 2 lines using the syntax `.sync` as we do in other parts of the code.

So instead of writing

```js
:visible="showQuickNavigationModal"
@update:visible="$emit('update:showQuickNavigationModal', false)"
```

you would write `:visible.sync="isVisible"` in the template and then create a computed property of `isVisible` with a get and a set

```js
isVisible: {
  get: ({ showQuickNavigationModal }) => showQuickNavigationModal,
  set(value) {
    this.$emit('update:showQuickNavigationModal', value);
  },
},
```